app/jobs/remove_old_throttles_job.rb

```diff
@@ -15,7 +15,7 @@ class RemoveOldThrottlesJob < ApplicationJob
 
   discard_on GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError
 
-  def perform(now, limit: 1000, total_limit: 100_000)
+  def perform(now, limit: 500, total_limit: 50_000)
```

Comment:
maybe even drop this more? since it may be a sparse table, the limit might be expensive to search for

Another idea: we could use a `find_in_batches` kind of approach that uses `id` key to search + page (because LIMIT / OFFSET is expensive)

Reply:
The index should help here, so I'm ok with trying this for now

Reply:
Oh, never mind, I understand now

Reply:
Actually, we don't use offset here, do we?

Going to roll forward with this for now.